src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1162,12 +1162,12 @@ public function get_source_info() {
 			}
 
 			$via = $this->get_label();
-			if ( in_array( $this->origin, array( 'meetup' ) ) ) {
+			if ( in_array( $this->origin, [ 'meetup' ] ) ) {
 				$via = '<a href="' . esc_url( $this->meta['source'] ) . '" target="_blank">' . esc_html( $via ) . '<span class="screen-reader-text">' . __( ' (opens in a new window)', 'the-events-calendar' ) . '</span></a>';
 			}
 		}
 
-		return array( 'title' => $title, 'via' => $via );
+		return [ 'title' => $title, 'via' => $via ];
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 9a4440e8bde820e43899e87d5ebab3a2</sub>